src/history/history-and-comments.component.tsx

```diff
@@ -226,7 +232,7 @@ const HistoryAndComments: React.FC<{
                   {dispense.performer && dispense.performer[0]?.actor?.display} {generateDispenseVerbiage(dispense)} -{' '}
                   {formatDatetime(parseDate(dispense.whenHandedOver))}
                 </h5>
-                <Tile className={styles.dispenseTile}>
+                <Tile className={styles.dispenseTile} data-floating-menu-container>
```

Comment:
What's the data attribute for?

Reply:
`OverflowMenu` component uses react portal to render and needs a parent with the `data-floating-menu-container` attribute [docs](https://react.carbondesignsystem.com/?path=/docs/components-overflowmenu--overview#data-floating-menu-container)